src/styles/card.scss

```diff
@@ -0,0 +1,108 @@
+.card {
+  padding: 32px 16px 16px;
+  width: 200px; // Фиксированная ширина
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  box-sizing: border-box;
+
+  &__imac {
+    padding: 0 3px;
```

Comment:
There should be no space before the value '3px'. According to the code style rules, there should be no spaces inside the parentheses.